src/query/vacuum.c

```diff
@@ -6031,14 +6021,6 @@ vacuum_log_add_dropped_file (THREAD_ENTRY * thread_p, const VFID * vfid, const O
 
   /* Initialize recovery data */
   VFID_COPY (&rcv_data.vfid, vfid);
-  if (class_oid != NULL)
-    {
-      COPY_OID (&rcv_data.class_oid, class_oid);
-    }
-  else
-    {
-      OID_SET_NULL (&rcv_data.class_oid);
```

Comment:
This initialization code is necessary. The rcv_data.class_oid will be referenced by REDO/UNDO functions. If there is an initialization value, then unexpected behavior will occur.

Reply:
I've removed the usage of the `class_oid`, please check `vacuum_rv_notify_dropped_file()` below

Reply:
I got it. However, I think it is safe to set it to null even if it is unused. (But, it depends on you. ^^)